d011-blackjack/Untitled-1.py

```diff
@@ -0,0 +1,128 @@
+#initially, randomly distribute two cards each to house and player 
+#if over 21, lose | if tie, house win | greater point wins otherwise
+#house has less than 16 points, add one card
+#player gets two cards, reveal
+#ask player if want another card
+#
+
+import random 
+card = ['A',
+         2,
+         3,
+         4,
+         5,
+         6,
+         7,
+         8,
+         9,
+        10,
+       'J',
+       'Q',
+       'K']
+    
+house_hand = []
+player_hand = []
+#random.shuffle(card)
+
+house_hand.append(random.choice(card))
+player_hand.append(random.choice(card))
+house_hand.append(random.choice(card))
+player_hand.append(random.choice(card))
+#print(house_hand)
+#print(player_hand)
+#print(player_hand[0])
+
+house_value = 0
+player_value = 0
+
+def Card_Value(card):
+    if card == 'A':
+        value = 11
+    elif card == 'J' or card == 'Q' or card == 'K':
+        value = 10
+    else:
+        value = card
+    return value
+
+def YesOrNo(user_choice):
+    return user_choice.lower() == "yes" or user_choice.lower() == "no"
+
+def AddCard(user_input, verify_input, invalid_input):
+    additional_card = input(user_input)
+    
+    while not verify_input(additional_card):
+        print(invalid_input)
+        additional_card = input(user_input)
+    return additional_card
+
+#def AceValue(user_hand, user_card_value):
+#    if ('A' in user_hand) and (user_card_value > 21):
+#        user_card_value -= 10
+#    return user_card_value
+
+def TotalValue(card_hand):
+    sum_value = 0
+    ace_count = 0
+    for card in card_hand:
+        sum_value += Card_Value(card)
+        if card == 'A':
+            ace_count +=1
+    while ace_count > 0 and sum_value > 21:
+        sum_value -=10
+        ace_count -=1
+    return sum_value
+
+
+#house_value = Card_Value(house_hand[0]) + Card_Value(house_hand[len(house_hand)-1])
+#player_value = Card_Value(player_hand[0]) + Card_Value(player_hand[len(player_hand)-1])
+player_value = TotalValue(player_hand)
+house_value = TotalValue(house_hand)
+print(f"Player: {player_hand} | {player_value}")
```

Comment:
In a normal game, the user can see _one_ card from the house before deciding if he/she wants more cards. Can we add the same option here?